Test.cpp

```diff
@@ -86,12 +98,12 @@ int main(int argc, char const *argv[])
   // ANS: Here it is an example of how to handle that case.
   vc::compiler_ptr_t clang = vc::make_compiler<vc::SystemCompilerOptimizer>(
                                           "llvm/clang",
-                                          "clang",
-                                          "opt",
+                                          CLANG_EXE_NAME,
+                                          OPT_EXE_NAME,
                                           ".",
-                                          "./test.log",
-                                          "/usr/bin",
-                                          "/usr/lib/llvm/bin"
+                                          "./test.log", 
+                                          LLVM_TOOLS_BINARY_DIR,
+                                          LLVM_TOOLS_BINARY_DIR
                                         );
 #if HAVE_CLANG_AS_LIB
   vc::compiler_ptr_t clangAsLib = vc::make_compiler<vc::ClangLibCompiler>(
```

Comment:
This is the part which probably breaks stuff in Travis CI. I have no clue on what could be wrong here
Reference:
https://app.travis-ci.com/github/Focshole/libVersioningCompiler/jobs/588014903#L622

Reply:
problem is the clang Driver that does not correctly passes options to the clang compiler. See `ClangLibCompiler::generateIR` around `ClangLibCompiler.cpp:118-151`

Reply:
Does this error happen only in the CI or also locally?
CI might be missing libClang or something like that